Buildings/Controls/OBC/ASHRAE/G36_PR1/TerminalUnits/ZoneModeAndSetPoints.mo

```diff
@@ -0,0 +1,366 @@
+    annotation (Line(points={{-59,-50},{-54,-50},{-54,-32},{-20,-32},{-20,-1}},
+      color={255,0,255}));
+  connect(TSetRooCooOn.y, reaRep.u)
+    annotation (Line(points={{-59,120},{-32,120}}, color={0,0,127}));
+  connect(reaRep.y, TSetZon.occCooSet)
+    annotation (Line(points={{-9,120},{16,120},{16,-2},{28,-2}}, color={0,0,127}));
+  connect(TSetRooHeaOn.y, reaRep1.u)
+    annotation (Line(points={{-59,90},{-42,90},{-42,92},{-32,92}},
+      color={0,0,127}));
+  connect(reaRep1.y, TSetZon.occHeaSet)
+    annotation (Line(points={{-9,92},{12,92},{12,-6},{28,-6}},
+      color={0,0,127}));
+  connect(TSetRooHeaOff.y, reaRep2.u)
+    annotation (Line(points={{-59,60},{-46,60},{-46,64},{-32,64}},
+      color={0,0,127}));
+  connect(reaRep2.y, TSetZon.unoHeaSet)
+    annotation (Line(points={{-9,64},{8,64},{8,-14},{28,-14}},
+      color={0,0,127}));
+  connect(TSetRooCooOff.y, reaRep3.u)
+    annotation (Line(points={{-59,30},{-48,30},{-48,36},{-32,36}},
+      color={0,0,127}));
+  connect(reaRep3.y, TSetZon.unoCooSet)
+    annotation (Line(points={{-9,36},{4,36},{4,-10},{28,-10}},
+      color={0,0,127}));
+  connect(opeModSel.yOpeMod, intRep.u)
+    annotation (Line(points={{-9,10},{10,10},{10,40},{26,40}},
+      color={255,127,0}));
+  connect(intRep.y, TSetZon.uOpeMod)
+    annotation (Line(points={{49,40},{60,40},{60,14},{20,14},{20,-30},{28,-30}},
+      color={255,127,0}));
+
+annotation (Diagram(coordinateSystem(extent={{-100,-100},{100,140}})),
+  Icon(graphics={Text(
+        extent={{-100,140},{98,102}},
+        textString="%name",
+        lineColor={0,0,255}),
+      Rectangle(
+        extent={{-100,-100},{100,100}},
+        lineColor={0,0,127},
+        fillColor={255,255,255},
+        fillPattern=FillPattern.Solid)}),
+Documentation(info="<html>
+<p>
+Block outputs zone setpoint temperature (<code>TCooSet</code>, <code>THeaSet</code>) 
+and system operation mode (<code>yOpeMod</code>). When operation mode is in freeze
+protection setback mode, it also outputs a level 3 freeze protection alarm 
+<code>yFreProSta</code>. The sequences are implemented according to ASHRAE
+Guideline 36, Part 5.B.3 and 5.C.6.
+</p>
+<p>The sequence consists of two subsequences. </p>
+<h4>a. Operation mode selector</h4>
+<p>
+The subsequence outputs 7 types system operation mode (occupied, warmup,
+cool-down, setback, freeze protection setback, setup, unoccupied) according 
+to current time, the time to next occupied hours <code>tNexOcc</code>, 
+current zone temperature <code>TZon</code>. See
+<a href=\"modelica://Buildings.Controls.OBC.ASHRAE.G36_PR1.Generic.SetPoints.OperationMode\">
+Buildings.Controls.OBC.ASHRAE.G36_PR1.Generic.SetPoints.OperationMode</a>.
+</p>
+<h4>b. Zone setpoint temperature reset</h4>
+<p>
+This sequence is implemented according to Part 5.B.3. It sets zone setpoint 
+according to the global giving setpoint, local setpoint adjustments, demand 
+limits adjustment, window status and occupancy stataus. See
```

Comment:
I would add "temperature" to zone setpoint. What is global giving setpoint? I looked in the referenced sequence, but it is not to be found.